docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -0,0 +1,63 @@
+# Moving your project to Angular-CLI
```

Comment:
I'm not sure it's such a great idea to have the new and old in the same folder. Are you sure?